### ISSUE TITLE
Invalid mapping

### ISSUE THREAD

**Issue by geoffrey-brier** [Sylius/Sylius#2376]
Hi guys,

If you run the `doctrine:schema:validate` command, it outputs:

```
[Mapping]  FAIL - The entity-class 'Sylius\Component\Product\Model\ProductTranslation' mapping is invalid:
* The association Sylius\Component\Product\Model\ProductTranslation#translatable refers to the inverse side field Sylius\Component\Product\Model\Product#translations which does not exist.
[Mapping]  FAIL - The entity-class 'Sylius\Component\Variation\Model\OptionTranslation' mapping is invalid:
* The association Sylius\Component\Variation\Model\OptionTranslation#translatable refers to the inverse side field Sylius\Component\Variation\Model\Option#translations which does not exist.
[Mapping]  FAIL - The entity-class 'Sylius\Component\Attribute\Model\AttributeTranslation' mapping is invalid:
* The association Sylius\Component\Attribute\Model\AttributeTranslation#translatable refers to the inverse side field Sylius\Component\Attribute\Model\Attribute#translations which does not exist.
[Mapping]  FAIL - The entity-class 'Sylius\Component\Shipping\Model\ShippingMethodTranslation' mapping is invalid:
* The association Sylius\Component\Shipping\Model\ShippingMethodTranslation#translatable refers to the inverse side field Sylius\Component\Shipping\Model\ShippingMethod#translations which does not exist.
[Mapping]  FAIL - The entity-class 'Sylius\Component\Taxonomy\Model\TaxonomyTranslation' mapping is invalid:
* The association Sylius\Component\Taxonomy\Model\TaxonomyTranslation#translatable refers to the inverse side field Sylius\Component\Taxonomy\Model\Taxonomy#translations which does not exist.
[Mapping]  FAIL - The entity-class 'Sylius\Component\Taxonomy\Model\TaxonTranslation' mapping is invalid:
* The association Sylius\Component\Taxonomy\Model\TaxonTranslation#translatable refers to the inverse side field Sylius\Component\Taxonomy\Model\Taxon#translations which does not exist.
```

Is it a known bug ?

**Comment by 4NN3CK73** [comment]
Same here ... any news? Hints, pointers?

**Comment by mtthb** [comment]
It appears to be fixed by #2377

**Comment by egeloen** [comment]
I confirm #2377 fixes the issue.
